config/puma.rb

```diff
@@ -0,0 +1,17 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 4)
+threads_count = Integer(ENV['MAX_THREADS'] || 1)
+threads threads_count, threads_count
+
+preload_app!
+
+rackup      DefaultRackup
+port        ENV['PORT']     || 3000
+environment ENV['RACK_ENV'] || 'development'
+
+on_worker_boot do
+  # Worker specific setup for Rails 4.1+
+  # See: https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#on-worker-boot
+  if defined?(ActiveRecord::Base)
```

Comment:
Hmm, though I can see it was in the unicorn config, so let's leave it for "changing less things"